src/histolab/slide.py

```diff
@@ -48,13 +64,23 @@ class Slide:
     """
 
     def __init__(
-        self, path: Union[str, pathlib.Path], processed_path: Union[str, pathlib.Path]
+        self,
+        path: Union[str, pathlib.Path],
+        processed_path: Union[str, pathlib.Path],
+        use_largeimage=False,
```

Comment:
This argument is missed in the docstring

Reply:
added.